src/Components/Trainers/Table/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import styles from './table.module.css';
+import ConfirmationModal from '../Modal';
+
+const Table = ({ data, deleteItem }) => {
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [selectedTrainer, setSelectedTrainer] = useState(null);
+
+  const handleDeleteConfirmation = () => {
+    if (selectedTrainer) {
+      deleteItem(selectedTrainer._id);
+      setShowConfirmationModal(false);
+      setSelectedTrainer(null);
+    }
+  };
+
+  const handleCancelConfirmation = () => {
+    setShowConfirmationModal(false);
+    setSelectedTrainer(null);
+  };
+
+  const openConfirmationModal = (trainer) => {
+    setSelectedTrainer(trainer);
+    setShowConfirmationModal(true);
+  };
+
+  return (
+    <div>
+      <table className={styles.tableContainer}>
+        <thead>
+          <tr>
+            <th>First Name</th>
+            <th>Last Name</th>
+            <th></th>
+            <th>Actions</th>
```

Comment:
*optional: you can remove the blank columns and unify the actions column. 
Example:
![ee](https://github.com/BaSP-m2023/nai-megarocket-app/assets/73259252/c8aa547b-1ad9-4092-8ba8-d9ac28396cc3)